src/modules/commentcounter.js

```diff
@@ -0,0 +1,2 @@
+const commentCounter = (list) => list.childElementCount + 1;
+export default commentCounter;
```

Comment:
Please consider rechecking the test for the `comment-counter` currently it's not working but the `movie counter` is working. The reason for this is that you might have accidentally added `+ 1` in the value which is causing the test to fail. I am attaching a screenshot for further clarification.  

## Screenshot of test:-
![image](https://user-images.githubusercontent.com/94223367/218160767-2ae598aa-4213-477d-9c33-c0eb12b3d22d.png)